vhdbuilder/scripts/windows/automate_helpers.sh

```diff
@@ -0,0 +1,102 @@
+        git commit -m "chore: windows release notes for release $1"
+    else
+        echo "to add git commit chore: bumping windows image version to $1"
+        git commit -m "chore: bumping windows image version to $1"
+    fi
+
+    git push -u origin $3 -f
+
+    set +x  # To avoid logging PAT during curl
+    
+    # check if the pull request already existed in case of validation failure below
+    # {
+    #     "message": "Validation Failed",
+    #     "errors": [
+    #     {
+    #       "resource": "PullRequest",
+    #       "code": "custom",
+    #       "message": "A pull request already exists for Azure:wsimageBump/230707."
+    #     }
+    #    ],
+    #    "documentation_url": "https://docs.github.com/rest/pulls/pulls#create-a-pull-request"
+    # }
+    if [[ $4 == "ReleaseNotes" ]]; then
+        result=$(curl -H "Authorization: token $github_access_token" \
+        -H "Accept: application/vnd.github.v3+json" \
+        "https://api.github.com/repos/Azure/AgentBaker/pulls?state=open&head=Azure:$3" |\
+        jq '.[] | select(.title == "chore: automated PR to update windows release notes for version '$1'")')
+        if [[ -n $result ]]; then
+            echo "Pull request at head '$3' with title \"chore: automated PR to update windows release notes for version '$1'\" existed already"
+            echo "Error: you cannot run image version bumping twice"
+            exit 1
+        else
+            curl -X POST \
+                -H "Authorization: token $github_access_token" \
+                -H "Content-Type: application/json" \
+                -d '{
+                    "title": "chore: automated PR to update windows release notes for version '$1'",
+                    "body": "This is an automated PR to '$4' for the windows VHD release with image version '$1'",
+                    "head": "'$3'",
+                    "base": "master"
+                }' \
+                https://api.github.com/repos/Azure/AgentBaker/pulls
+        fi
+    else
+        result=$(curl -H "Authorization: token $github_access_token" \
+        -H "Accept: application/vnd.github.v3+json" \
+        "https://api.github.com/repos/Azure/AgentBaker/pulls?state=open&head=Azure:$3" |\
+        jq '.[] | select(.title == "chore: automated PR to bump windows image version to '$1'")')
+        if [[ -n $result ]]; then
+            echo "Pull request at head '$3' with title \"chore: automated PR to bump windows image version to '$1'\" existed already"
+            echo "Error: you cannot run image version bumping twice"
+            exit 1
+        else
+            curl -X POST \
+                -H "Authorization: token $github_access_token" \
+                -H "Content-Type: application/json" \
+                -d '{
+                    "title": "chore: automated PR to bump windows image version to '$1'",
+                    "body": "This is an automated PR to '$4' for the windows VHD release with image version '$1'",
```

Comment:
Update the body to match the title?

Reply:
Updated. nit: Title is subject to change e.g. https://github.com/Azure/AgentBaker/pull/3517